scripts/publish-wl-local.sh

```
@@ -0,0 +1,257 @@
+VERSION="${1:-}"
+[ -z "$VERSION" ] && fail "Usage: bash scripts/publish-wl-local.sh <version>  e.g. 2.1.0"
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$REPO_ROOT"
+
+# ── Cleanup .speakeasy on exit ──────────────────────────────────────────────
+_cleanup() {
+  echo ""
+  step "Restoring .speakeasy/ to original state"
+  git checkout -- .speakeasy/gen .speakeasy/workflow.yaml .speakeasy/overlays 2>/dev/null \
+    && ok ".speakeasy/ restored" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)